traitsui/tests/test_ui.py

```diff
@@ -162,25 +164,25 @@ def test_destroy_after_ok_wx(self):
         import wx
 
         foo = FooDialog()
-        ui = foo.edit_traits()
+        with create_ui(foo) as ui:
```

Comment:
Doesn't this test (and the one below) need `store_exceptions_on_all_threads()`? Other tests might need it as well, but this might be better done in a separate PR

Reply:
Agreed.

Reply:
Opened https://github.com/enthought/traitsui/issues/876